spec/requests/health_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Health Controller', type: :request do
```

Comment:
I am not sure about the test class name and also the controller test, following [Compass](https://nimblehq.co/compass/development/code-conventions/ruby/rspec/#convention-13) we don't write controller test at all 🤔

Reply:
Normally, we do not write many controller tests. We prefer writing request tests or UI tests.

So since it is an API, we can name it like the following:

```suggestion
RSpec.describe 'Health Endpoint', type: :request do
```